x-pack/plugins/cloud_security_posture/server/routes/benchmark_rules/bulk_action/bulk_action.ts

```diff
@@ -16,6 +16,29 @@ import { CspRouter } from '../../../types';
 import { CSP_BENCHMARK_RULES_BULK_ACTION_ROUTE_PATH } from '../../../../common/constants';
 import { bulkActionBenchmarkRulesHandler } from './v1';
 
+/**
+	This API allows bulk actions (mute or unmute) on CSP benchmark rules.
```

Comment:
I think it's a really good idea to document our APIs, but should we approach it more consistently with OpenAPI schema which other plugins use? There is also a tool to autogenerate types from OpenAPI schemas recenty introduced by the security team https://github.com/elastic/kibana/blob/main/packages/kbn-openapi-generator/README.md
Not for this PR definetly, but smth to consider in the future. The problem with defining the specs in the comments that they are hard to keep up to date. Eg. this comment is already out of date as the request body now is 
```
rules: [
	    {
	      rule_id: string;            // Unique identifier for the rule
	    },
	    // ... (additional benchmark rules)
	  ];
	}
```

Reply:
I don't find it right now, but Kfir mentioned that we have a ticket in the backlog for adding our APIs to OpenAPI.

Reply:
yeah, can't find it either. I will create a placeholder for it so we could discuss the priority of it

Reply:
https://github.com/elastic/kibana/issues/173830